### ISSUE TITLE
User account not found in MSAL cache on Linux

### ISSUE THREAD

**Issue by cheenamalhotra** [microsoft/azuredatastudio#23688]
Azure Data Studio Version: 1.44.1
Commit: 8f53a316fa00a98264f1ab119641cd540b5af25c
Date: 2023-06-01T02:10:44.473Z
OS: Linux x64 6.4.1-arch1-1

I tried unlinking account from Azure Data Studio, removing directories `~/.config/azuredatastudio/Azure Accounts` and `~/.config/azuredatastudio/CachedData`, and logging back in via private chrome window to ensure there is no cached login, but to no avail, I still get the error when I try to connect.

Any more tips how could I resolve the problem, while waiting for the July release?

Stacktrace:

```
Microsoft.Data.SqlClient.SqlException (0x80131904): User account '[redacted]@[redacted]' not found in MSAL cache, please add linked account or refresh account credentials.
 ---> System.Exception: User account '[redacted]@[redacted]' not found in MSAL cache, please add linked account or refresh account credentials.
   at Microsoft.SqlTools.Authentication.Authenticator.GetTokenAsync(AuthenticationParams params, CancellationToken cancellationToken) in /_/src/Microsoft.SqlTools.Authentication/Authenticator.cs:line 129
   at Microsoft.SqlTools.Authentication.Sql.AuthenticationProvider.AcquireTokenAsync(SqlAuthenticationParameters parameters) in /_/src/Microsoft.SqlTools.Authentication/Sql/AuthenticationProvider.cs:line 90
   at Microsoft.Data.SqlClient.SqlInternalConnectionTds.<>c__DisplayClass147_1.<<GetFedAuthToken>b__1>d.MoveNext()
--- End of stack trace from previous location ---
   at Microsoft.Data.SqlClient.SqlInternalConnectionTds.GetFedAuthToken(SqlFedAuthInfo fedAuthInfo)
   at Microsoft.Data.SqlClient.SqlInternalConnectionTds.GetFedAuthToken(SqlFedAuthInfo fedAuthInfo)
   at Microsoft.Data.SqlClient.SqlInternalConnectionTds.OnFedAuthInfo(SqlFedAuthInfo fedAuthInfo)
   at Microsoft.Data.SqlClient.TdsParser.TryRun(RunBehavior runBehavior, SqlCommand cmdHandler, SqlDataReader dataStream, BulkCopySimpleResultSet bulkCopyHandler, TdsParserStateObject stateObj, Boolean& dataReady)
   at Microsoft.Data.SqlClient.TdsParser.Run(RunBehavior runBehavior, SqlCommand cmdHandler, SqlDataReader dataStream, BulkCopySimpleResultSet bulkCopyHandler, TdsParserStateObject stateObj)
   at Microsoft.Data.SqlClient.SqlInternalConnectionTds.CompleteLogin(Boolean enlistOK)
   at Microsoft.Data.SqlClient.SqlInternalConnectionTds.AttemptOneLogin(ServerInfo serverInfo, String newPassword, SecureString newSecurePassword, Boolean ignoreSniOpenTimeout, TimeoutTimer timeout, Boolean withFailover)
   at Microsoft.Data.SqlClient.SqlInternalConnectionTds.LoginNoFailover(ServerInfo serverInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString connectionOptions, SqlCredential credential, TimeoutTimer timeout)
   at Microsoft.Data.SqlClient.SqlInternalConnectionTds.OpenLoginEnlist(TimeoutTimer timeout, SqlConnectionString connectionOptions, SqlCredential credential, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance)
   at Microsoft.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, Boolean applyTransientFaultHandling, String accessToken, DbConnectionPool pool)
   at Microsoft.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   at Microsoft.Data.ProviderBase.DbConnectionFactory.CreateNonPooledConnection(DbConnection owningConnection, DbConnectionPoolGroup poolGroup, DbConnectionOptions userOptions)
   at Microsoft.Data.ProviderBase.DbConnectionFactory.<>c__DisplayClass48_0.<CreateReplaceConnectionContinuation>b__0(Task`1 _)
   at System.Threading.Tasks.ContinuationResultTaskFromResultTask`2.InnerInvoke()
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
--- End of stack trace from previous location ---
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
   at System.Threading.Tasks.Task.ExecuteWithThreadLocal(Task& currentTaskSlot, Thread threadPoolThread)
--- End of stack trace from previous location ---
   at Microsoft.SqlTools.ServiceLayer.Connection.ReliableConnection.ReliableSqlConnection.<>c__DisplayClass30_0.<<OpenAsync>b__0>d.MoveNext() in /_/src/Microsoft.SqlTools.ManagedBatchParser/ReliableConnection/ReliableSqlConnection.cs:line 313
--- End of stack trace from previous location ---
   at Microsoft.SqlTools.ServiceLayer.Connection.ConnectionService.TryOpenConnection(ConnectionInfo connectionInfo, ConnectParams connectionParams) in /_/src/Microsoft.SqlTools.ServiceLayer/Connection/ConnectionService.cs:line 692
ClientConnectionId:[redacted]
```

[1-Azure-Acount-redacted.log](https://github.com/microsoft/azuredatastudio/files/11947804/1-Azure-Acount-redacted.log)

_Originally posted by @mishioo and @Anganthier in https://github.com/microsoft/azuredatastudio/issues/23286#issuecomment-1619927246_

**Comment by cheenamalhotra** [comment]
Hi @mishioo and @Anganthier

I was able to reproduce the error on Ubuntu 22 with stable build, and also verified that using [Azure Data Studio - Insiders](https://github.com/microsoft/azuredatastudio#try-out-the-latest-insiders-build-from-main-branch) can get you unblocked and resolve the issue, please install the same.

This issue will be resolved in the next release. Let us know if you continue to face any issues with insiders build.

**Comment by Anganthier** [comment]
The problem still persists. 
I already tried to deleteing .config/azuredatastudio first. Didn't change the outcome. I'll run the trace next.

Version: 1.45.0-insider
Commit: 55b8e3f6372175032393418d64301eb5c301b1c0
Date: 2023-07-06T05:55:20.906Z
VS Code: 1.79.0
Electron: 22.3.14
Chromium: 108.0.5359.215
Node.js: 16.17.1
V8: 10.8.168.25-electron.0
OS: Linux x64 5.15.0-76-generic

```
Microsoft.Data.SqlClient.SqlException (0x80131904): User account '[redacted]' not found in MSAL cache, please add linked account or refresh account credentials.
 ---> System.Exception: User account '[redacted]' not found in MSAL cache, please add linked account or refresh account credentials.
   at Microsoft.SqlTools.Authentication.Authenticator.GetTokenAsync(AuthenticationParams params, CancellationToken cancellationToken) in /_/src/Microsoft.SqlTools.Authentication/Authenticator.cs:line 129
   at Microsoft.SqlTools.Authentication.Sql.AuthenticationProvider.AcquireTokenAsync(SqlAuthenticationParameters parameters) in /_/src/Microsoft.SqlTools.Authentication/Sql/AuthenticationProvider.cs:line 90
   at Microsoft.Data.SqlClient.SqlInternalConnectionTds.<>c__DisplayClass147_1.<<GetFedAuthToken>b__1>d.MoveNext()
--- End of stack trace from previous location ---
   at Microsoft.Data.SqlClient.SqlInternalConnectionTds.GetFedAuthToken(SqlFedAuthInfo fedAuthInfo)
   at Microsoft.Data.SqlClient.SqlInternalConnectionTds.GetFedAuthToken(SqlFedAuthInfo fedAuthInfo)
   at Microsoft.Data.SqlClient.SqlInternalConnectionTds.OnFedAuthInfo(SqlFedAuthInfo fedAuthInfo)
   at Microsoft.Data.SqlClient.TdsParser.TryRun(RunBehavior runBehavior, SqlCommand cmdHandler, SqlDataReader dataStream, BulkCopySimpleResultSet bulkCopyHandler, TdsParserStateObject stateObj, Boolean& dataReady)
   at Microsoft.Data.SqlClient.TdsParser.Run(RunBehavior runBehavior, SqlCommand cmdHandler, SqlDataReader dataStream, BulkCopySimpleResultSet bulkCopyHandler, TdsParserStateObject stateObj)
   at Microsoft.Data.SqlClient.SqlInternalConnectionTds.CompleteLogin(Boolean enlistOK)
   at Microsoft.Data.SqlClient.SqlInternalConnectionTds.AttemptOneLogin(ServerInfo serverInfo, String newPassword, SecureString newSecurePassword, Boolean ignoreSniOpenTimeout, TimeoutTimer timeout, Boolean withFailover)
   at Microsoft.Data.SqlClient.SqlInternalConnectionTds.LoginNoFailover(ServerInfo serverInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString connectionOptions, SqlCredential credential, TimeoutTimer timeout)
   at Microsoft.Data.SqlClient.SqlInternalConnectionTds.OpenLoginEnlist(TimeoutTimer timeout, SqlConnectionString connectionOptions, SqlCredential credential, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance)
   at Microsoft.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, Boolean applyTransientFaultHandling, String accessToken, DbConnectionPool pool)
   at Microsoft.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   at Microsoft.Data.ProviderBase.DbConnectionFactory.CreateNonPooledConnection(DbConnection owningConnection, DbConnectionPoolGroup poolGroup, DbConnectionOptions userOptions)
   at Microsoft.Data.ProviderBase.DbConnectionFactory.<>c__DisplayClass48_0.<CreateReplaceConnectionContinuation>b__0(Task`1 _)
   at System.Threading.Tasks.ContinuationResultTaskFromResultTask`2.InnerInvoke()
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
--- End of stack trace from previous location ---
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
   at System.Threading.Tasks.Task.ExecuteWithThreadLocal(Task& currentTaskSlot, Thread threadPoolThread)
--- End of stack trace from previous location ---
   at Microsoft.SqlTools.ServiceLayer.Connection.ReliableConnection.ReliableSqlConnection.<>c__DisplayClass30_0.<<OpenAsync>b__0>d.MoveNext() in /_/src/Microsoft.SqlTools.ManagedBatchParser/ReliableConnection/ReliableSqlConnection.cs:line 313
--- End of stack trace from previous location ---
   at Microsoft.SqlTools.ServiceLayer.Connection.ConnectionService.TryOpenConnection(ConnectionInfo connectionInfo, ConnectParams connectionParams) in /_/src/Microsoft.SqlTools.ServiceLayer/Connection/ConnectionService.cs:line 697
ClientConnectionId:1348735a-b3eb-4787-a813-0b759eec489c
```

**Comment by Anganthier** [comment]
Just saw, that this issue is closed - new ticket or reopen?

**Comment by cheenamalhotra** [comment]
I can re-open the same for investigation - please upload full logs as per above comment.

**Comment by Anganthier** [comment]
As requested:
[console.log](https://github.com/microsoft/azuredatastudio/files/11994825/console.log)
[20230708T105808.zip](https://github.com/microsoft/azuredatastudio/files/11994826/20230708T105808.zip)

**Comment by cheenamalhotra** [comment]
Hi @Anganthier 

The logs are mostly empty other than renderer and it seems like there is a token acquired but it lacks expiry date.  To investigate more, can you please enable below logging levels in settings.json file:

```
"mssql:tracingLevel": "Verbose",
"azure.loggingLevel": "Verbose"
```

Once you enable these logs - please reload ADS and try reproducing the error again. Please upload logs again with verbose logging.

**Comment by piyushpaliwal** [comment]
I am facing the same issue. Here is the setup I am currently using
- Version: 1.46.0-insider
- OS: Debian 12

Tried the following
- Clear token cache from Azure Data Studio
- Remove All saved Accounts
- Manually deleting `Azure Accounts` and `CachedData` folder inside `~/.config/azuredatastudio`

**Comment by cheenamalhotra** [comment]
I also tested with [latest insiders build](https://github.com/microsoft/azuredatastudio#try-out-the-latest-insiders-build-from-main-branch) on Debian 12 and Ubuntu 22 and I can't repro issue with AAD auth/server logins.

![image](https://github.com/microsoft/azuredatastudio/assets/13396919/04bdf716-440f-46b4-8dd4-c3d61af7d331)

<img src="https://github.com/microsoft/azuredatastudio/assets/13396919/206fd4bd-76d5-4197-a496-168de081b3fb" width=400 />

Please upload logs for further investigation.

**Comment by bard83** [comment]
Moved my comment here. 

Same issue.

Below the exception and the logs.

```
Microsoft.Data.SqlClient.SqlException (0x80131904): User account '*****@*******' not found in MSAL cache, please add linked account or refresh account credentials.
 ---> System.Exception: User account ''*****@*******' not found in MSAL cache, please add linked account or refresh account credentials.
   at Microsoft.SqlTools.Authentication.Authenticator.GetTokenAsync(AuthenticationParams params, CancellationToken cancellationToken) in /_/src/Microsoft.SqlTools.Authentication/Authenticator.cs:line 129
   at Microsoft.SqlTools.Authentication.Sql.AuthenticationProvider.AcquireTokenAsync(SqlAuthenticationParameters parameters) in /_/src/Microsoft.SqlTools.Authentication/Sql/AuthenticationProvider.cs:line 90
   at Microsoft.Data.SqlClient.SqlInternalConnectionTds.<>c__DisplayClass147_1.<<GetFedAuthToken>b__1>d.MoveNext()
--- End of stack trace from previous location ---
   at Microsoft.Data.SqlClient.SqlInternalConnectionTds.GetFedAuthToken(SqlFedAuthInfo fedAuthInfo)
   at Microsoft.Data.SqlClient.SqlInternalConnectionTds.GetFedAuthToken(SqlFedAuthInfo fedAuthInfo)
   at Microsoft.Data.SqlClient.SqlInternalConnectionTds.OnFedAuthInfo(SqlFedAuthInfo fedAuthInfo)
   at Microsoft.Data.SqlClient.TdsParser.TryRun(RunBehavior runBehavior, SqlCommand cmdHandler, SqlDataReader dataStream, BulkCopySimpleResultSet bulkCopyHandler, TdsParserStateObject stateObj, Boolean& dataReady)
   at Microsoft.Data.SqlClient.TdsParser.Run(RunBehavior runBehavior, SqlCommand cmdHandler, SqlDataReader dataStream, BulkCopySimpleResultSet bulkCopyHandler, TdsParserStateObject stateObj)
   at Microsoft.Data.SqlClient.SqlInternalConnectionTds.CompleteLogin(Boolean enlistOK)
   at Microsoft.Data.SqlClient.SqlInternalConnectionTds.AttemptOneLogin(ServerInfo serverInfo, String newPassword, SecureString newSecurePassword, Boolean ignoreSniOpenTimeout, TimeoutTimer timeout, Boolean withFailover)
   at Microsoft.Data.SqlClient.SqlInternalConnectionTds.LoginNoFailover(ServerInfo serverInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString connectionOptions, SqlCredential credential, TimeoutTimer timeout)
   at Microsoft.Data.SqlClient.SqlInternalConnectionTds.OpenLoginEnlist(TimeoutTimer timeout, SqlConnectionString connectionOptions, SqlCredential credential, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance)
   at Microsoft.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, Boolean applyTransientFaultHandling, String accessToken, DbConnectionPool pool)
   at Microsoft.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   at Microsoft.Data.ProviderBase.DbConnectionFactory.CreateNonPooledConnection(DbConnection owningConnection, DbConnectionPoolGroup poolGroup, DbConnectionOptions userOptions)
   at Microsoft.Data.ProviderBase.DbConnectionFactory.<>c__DisplayClass48_0.<CreateReplaceConnectionContinuation>b__0(Task`1 _)
   at System.Threading.Tasks.ContinuationResultTaskFromResultTask`2.InnerInvoke()
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
--- End of stack trace from previous location ---
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
   at System.Threading.Tasks.Task.ExecuteWithThreadLocal(Task& currentTaskSlot, Thread threadPoolThread)
--- End of stack trace from previous location ---
   at Microsoft.SqlTools.ServiceLayer.Connection.ReliableConnection.ReliableSqlConnection.<>c__DisplayClass30_0.<<OpenAsync>b__0>d.MoveNext() in /_/src/Microsoft.SqlTools.ManagedBatchParser/ReliableConnection/ReliableSqlConnection.cs:line 313
--- End of stack trace from previous location ---
   at Microsoft.SqlTools.ServiceLayer.Connection.ConnectionService.TryOpenConnection(ConnectionInfo connectionInfo, ConnectParams connectionParams) in /_/src/Microsoft.SqlTools.ServiceLayer/Connection/ConnectionService.cs:line 692
ClientConnectionId:0d42e9d9-2cdf-49aa-bb8e-aecc1963f764
```

[1-Azure Accounts.log](https://github.com/microsoft/azuredatastudio/files/12039103/1-Azure.Accounts.log)

**Comment by bard83** [comment]
[console.log](https://github.com/microsoft/azuredatastudio/files/12044409/console.log)
[20230713T224626.zip](https://github.com/microsoft/azuredatastudio/files/12044411/20230713T224626.zip)

**Comment by cheenamalhotra** [comment]
I see these errors in exthost.log file btw.. 

```
[2023-07-13 22:49:05.925] [exthost] [debug] ProxyResolver#resolveProxy https://login.microsoftonline.com/common/discovery/instance?api-version=1.1&authorization_endpoint=https://login.microsoftonline.com/02dca5ee-e24f-4cee-ab0a-1d1054bebd16/oauth2/v2.0/authorize DIRECT 
Error: Error for stack trace
	at resolveProxy (/usr/share/azuredatastudio/resources/app/node_modules.asar/vscode-proxy-agent/out/index.js:105:69)
	at /usr/share/azuredatastudio/resources/app/node_modules.asar/vscode-proxy-agent/out/index.js:294:77
	at new Promise (<anonymous>)
	at _PacProxyAgent.resolveP [as resolver] (/usr/share/azuredatastudio/resources/app/node_modules.asar/vscode-proxy-agent/out/index.js:294:54)
	at _PacProxyAgent.<anonymous> (/usr/share/azuredatastudio/resources/app/node_modules.asar/vscode-proxy-agent/out/agent.js:70:37)
	at Generator.next (<anonymous>)
	at /usr/share/azuredatastudio/resources/app/node_modules.asar/vscode-proxy-agent/out/agent.js:8:71
	at new Promise (<anonymous>)
	at __awaiter (/usr/share/azuredatastudio/resources/app/node_modules.asar/vscode-proxy-agent/out/agent.js:4:12)
	at _PacProxyAgent.callback (/usr/share/azuredatastudio/resources/app/node_modules.asar/vscode-proxy-agent/out/agent.js:52:16)
	at _PacProxyAgent.addRequest (/usr/share/azuredatastudio/resources/app/node_modules.asar/agent-base/dist/src/index.js:184:38)
	at new ClientRequest (node:_http_client:305:16)

[2023-07-13 22:49:06.329] [exthost] [debug] ProxyResolver#resolveProxy cached https://login.microsoftonline.com/02dca5ee-e24f-4cee-ab0a-1d1054bebd16/oauth2/v2.0/token DIRECT 
Error: Error for stack trace
	at resolveProxy (/usr/share/azuredatastudio/resources/app/node_modules.asar/vscode-proxy-agent/out/index.js:105:69)
	at /usr/share/azuredatastudio/resources/app/node_modules.asar/vscode-proxy-agent/out/index.js:294:77
	at new Promise (<anonymous>)
	at _PacProxyAgent.resolveP [as resolver] (/usr/share/azuredatastudio/resources/app/node_modules.asar/vscode-proxy-agent/out/index.js:294:54)
	at _PacProxyAgent.<anonymous> (/usr/share/azuredatastudio/resources/app/node_modules.asar/vscode-proxy-agent/out/agent.js:70:37)
	at Generator.next (<anonymous>)
	at /usr/share/azuredatastudio/resources/app/node_modules.asar/vscode-proxy-agent/out/agent.js:8:71
	at new Promise (<anonymous>)
	at __awaiter (/usr/share/azuredatastudio/resources/app/node_modules.asar/vscode-proxy-agent/out/agent.js:4:12)
	at _PacProxyAgent.callback (/usr/share/azuredatastudio/resources/app/node_modules.asar/vscode-proxy-agent/out/agent.js:52:16)
	at _PacProxyAgent.addRequest (/usr/share/azuredatastudio/resources/app/node_modules.asar/agent-base/dist/src/index.js:184:38)
	at new ClientRequest (node:_http_client:305:16)
	at request (node:https:353:10)
	at Object.patched [as request] (/usr/share/azuredatastudio/resources/app/node_modules.asar/vscode-proxy-agent/out/index.js:298:24)
	at /usr/share/azuredatastudio/resources/app/extensions/azurecore/node_modules/@azure/msal-node/dist/msal-node.cjs.development.js:322:27
```

Are you missing any proxy settings?

**Comment by bard83** [comment]
I'm not behind any proxy server. I'm tunnelling the connection using ssh through bastion on azure.

**Comment by bard83** [comment]
Mitigation suggested [here](https://github.com/microsoft/vscode-mssql/issues/17744#issuecomment-1634936754) helped.

**Comment by cssuh** [comment]
Hi, can you please install the latest [insiders](https://github.com/microsoft/azuredatastudio#try-out-the-latest-insiders-build-from-main-branch) and run the `Azure Accounts: Clear Azure Account Token Cache` command? We just pushed out a [fix](https://github.com/microsoft/azuredatastudio/pull/23875) for this issue.

**Comment by bard83** [comment]
It still does not work.

My test:
The inserders version that you suggested has been installed. The configuration which mitigated the problem has been set back. Azure Account has been cleared. Created Connection and tried con connect. 


[console-insiders.log](https://github.com/microsoft/azuredatastudio/files/12060921/console-insiders.log)

[20230715T101037.zip](https://github.com/microsoft/azuredatastudio/files/12060919/20230715T101037.zip)

**Comment by Anganthier** [comment]
Hi,

it works for me.

I removed the current installation of "azure data studio - insider", 
deleted ~/.config/azuredatastudio/, 
deleted ~/.azuredatastudio-insiders, 
installed the azure data studio,
added my ms account
added the connection to the sql server
-> it works, no error showing up

Thanks,
Anganthier

**Comment by bard83** [comment]
I messed things a little bit up. My bad.

Cleaned up the installations and re-installed the insider version.

It worked. Thanks.

**Comment by kisantia** [comment]
fix has been verified
